airbyte-persistence/job-persistence/src/main/java/io/airbyte/persistence/job/tracker/JobTracker.java

```diff
@@ -246,7 +246,7 @@ private static Map<String, Object> configToMetadata(final JsonNode config, final
       // * Otherwise, do some basic conversions to value-ish data.
       // It would be a weird thing to declare const: null, but in that case we don't want to report null
       // anyway, so explicitly use hasNonNull.
-      return Jsons.flatten(config);
+      return Jsons.flatten(config, false);
```

Comment:
this seems unnecessary given that there is a default method that you added for backwards compatibility

Reply:
Up to you all - might be better to be explicit than implicit? I have no strong feelings either way

Reply:
Addressed in [0e55dd5](https://github.com/airbytehq/airbyte/pull/17273/commits/0e55dd516f7fb7657ab23150d527eefd92fd358f)